.gitignore

```diff
@@ -0,0 +1,9 @@
+# documentation artifacts
```

Comment:
Please remove the `.gitignore` file from Git tracking.

Reply:
I'd like to add that we have added the .gitignore with similar contents to most other repositories. This was the first concern raised about a .gitignore. As a result, I would also like to understand your views on this.

Reply:
In my opinion a `.gitignore` is developer specific since each developer uses a different development tool, the resulting artifacts vary. For instance I use VS code which create many file that need to be ignored specific to my dev env. Thus I can have a local `.gitignore` and add `.gitignore` to it followed by all the files I want to keep out of source control. This means that when I pull MIOpen into my local copy, there is no conflict. However, if we add a gitignore to the repo every time I do a pull there will be a merge conflict which would need to be resolved. 

If someone accidentally adds artifacts from their build, they will have to be reviewed and would be highlighted.

Reply:
Done in [remove .gitignore](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/2b5535b784a9d0331f0dc8b861f0b96823d54c71)